api/HTMLAreaElement.json

```diff
@@ -574,54 +574,6 @@
             "deprecated": false
           }
         }
-      },
-      "type": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLAreaElement/type",
```

Comment:
https://developer.mozilla.org/docs/Web/API/HTMLAreaElement/type does not exist, so nothing to clean up there.